facebook/react-native
# macOS : request interactive input when killing the packager

Do not request interactive input when running on a CI

When I'm running the build on my CI server, many Terminal windows are remaining on the server because of the `read` command.

This PR prevent this behaviour when the `CI` environment variable is defined so that it doesn't alter the behaviour on local machines.

## Test Plan

 * Run a build when the packager killing step at the end
 * Ensure no window remain opened on the server (macOS)

## Release Notes

[IOS] [MINOR] [launchPackager.command] - Do not request interactive input when killing packager on a CI server
